### PR TITLE
[nl]: Use T form in Dutch translations

### PR DESCRIPTION
**Pull request by standaniels**
This PR uses the informal T form instead of the formal V form in the Dutch translations.

In Dutch it's becoming less and less common to use the V form when addressing someone you haven't met. I noticed that currently both T and V forms are used, so I decided to use the T form everywhere.

I also fixed some errors while I was at it:
- Change `BTW` to `btw` ([source](https://onzetaal.nl/taalloket/btw-btw))
- Remove spaces of words that are 'whole' (e.g. `QR-code` instead of `QR code`)
- Fix "d/t-errors"
- Various other fixes

### CODE REVIEW

**Comment by andrey-helldar** [comment]
@WhereIsLucas, can you check this PR?

File: locales/nl/json.json [hunk 1]
```diff
@@ -326,7 +326,7 @@
     "Finish enabling two factor authentication.": "Inschakelen van tweestapsverificatie afronden.",
     "Finland": "Finland",
     "Florida": "Florida",
-    "For security reasons your new card will be charged a minimal fee upon registration.": "Om veiligheidsredenen wordt uw nieuwe kaart bij registratie een minimale vergoeding in rekening gebracht.",
+    "For security reasons your new card will be charged a minimal fee upon registration.": "Om veiligheidsredenen word je nieuwe kaart bij registratie een minimale vergoeding in rekening gebracht.",
```
Comment (ronn0) [inline locales/nl/json.json:329]: This one is incorrect. It should be `wordt` because the subject is `je nieuwe kaart`

Comment (ronn0) [reply locales/nl/json.json:329]: Besides that the sentence isn't right, this one is better I guess:
`"Om veiligheidsredenen wordt bij registratie van je nieuwe kaart een minimale vergoeding in rekening gebracht.`

File: locales/nl/json.json [hunk 2]
```diff
@@ -1071,69 +1071,69 @@
...
     "Yes, Leave": "Ja, vertrek",
     "Yes, Remove": "Ja, verwijderen",
-    "You": "U",
+    "You": "Je",
     "You are already subscribed.": "Je bent al ingeschreven.",
-    "You are currently subscribed to the :planName plan.": "U bent momenteel geabonneerd op het :planName-abonnement.",
+    "You are currently subscribed to the :planName plan.": "Je bent momenteel geabonneerd op het :planName-abonnement.",
     "You are currently within your free trial period. Your trial will expire on :date.": "Je maakt nu gebruik van een gratis proefabonnement. Je proefabonnement verloopt op :date.",
-    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "U bevindt zich momenteel in uw gratis proefperiode. Uw proefperiode verloopt op :date. Als u een nieuw abonnement start, wordt uw proefperiode beëindigd.",
+    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "Je bevindt zich momenteel in je gratis proefperiode. Je proefperiode verloopt op :date. Als je een nieuw abonnement start, word je proefperiode beëindigd.",
```
Comment (ronn0) [inline locales/nl/json.json:1078]: This one should be `wordt`

Comment (standaniels) [reply locales/nl/json.json:1078]: Oops, you're absolutely right.

File: locales/nl/json.json (hunk 1 above)
Comment (standaniels) [reply locales/nl/json.json:329]: Oops, you're absolutely right. And nice suggestion, I will commit the change.

**Comment by andrey-helldar** [comment]
@WhereIsLucas, ping

**Review by ronn0 — APPROVED** [review]

**Review by andrey-helldar — APPROVED** [review]
Since @WhereIsLucas is busy, I will accept these changes based on the information from @ronn0.

@standaniels, @ronn0, Dankjewel!

File: locales/nl/json.json [hunk 3]
```diff
@@ -1071,69 +1071,69 @@
...
     "Yes, Leave": "Ja, vertrek",
     "Yes, Remove": "Ja, verwijderen",
-    "You": "U",
+    "You": "Je",
     "You are already subscribed.": "Je bent al ingeschreven.",
-    "You are currently subscribed to the :planName plan.": "U bent momenteel geabonneerd op het :planName-abonnement.",
+    "You are currently subscribed to the :planName plan.": "Je bent momenteel geabonneerd op het :planName-abonnement.",
     "You are currently within your free trial period. Your trial will expire on :date.": "Je maakt nu gebruik van een gratis proefabonnement. Je proefabonnement verloopt op :date.",
-    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "U bevindt zich momenteel in uw gratis proefperiode. Uw proefperiode verloopt op :date. Als u een nieuw abonnement start, wordt uw proefperiode beëindigd.",
+    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "Je bevindt zich momenteel in je gratis proefperiode. Je proefperiode verloopt op :date. Als je een nieuw abonnement start, wordt je proefperiode beëindigd.",
```
Comment (devfrey) [inline locales/nl/json.json:1078]: "Je bevindt zich momenteel in je gratis proefperiode." is incorrect. It should be "Je bevindt **je** momenteel in je gratis proefperiode."

Comment (standaniels) [reply locales/nl/json.json:1078]: Fixed in https://github.com/Laravel-Lang/lang/pull/3255.

**Pull request merged**